### PR TITLE
Full height: single image / single copy

### PR DESCRIPTION
**Pull request by Heleen-cr**
Preview environment here: http://pr-622-3qjmcyy-3g6y4v7pqt6nk.eu.platform.sh/

### CODE REVIEW

File: sass/themes/cr/2019/components/full-height-single-image-single-copy/_full-height-single-image-single-copy.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        overflow: hidden;
+        height: inherit;
+        // position image absolutely so height of section will be based on the text
+        img {
+          @include no-objectfit;
+          position: absolute;
+          width: 100%;
+          height: 100%;
+          object-fit: cover;
+
```
Comment (AndyEPhipps) [inline sass/themes/cr/2019/components/full-height-single-image-single-copy/_full-height-single-image-single-copy.scss]: too many newlines

File: sass/themes/cr/2019/components/full-height-single-image-single-copy/full-height-single-image-single-copy.twig [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<section class="paragraph--full-height-single-image-single-copy {{modifier_class}} bg--coral">
+  <div class="fhsisc__image-wrapper objectfit">
+     <img src="kss-assets/images/spice_3-4_2x.jpg">
```
Comment (AndyEPhipps) [inline sass/themes/cr/2019/components/full-height-single-image-single-copy/full-height-single-image-single-copy.twig]: Was gonna suggest just using the picture element to load a 4:3 image for mobile if you've got time

File: sass/themes/cr/2019/components/full-height-single-image-single-copy/_full-height-single-image-single-copy.scss (hunk 1 above)
Comment (Heleen-cr) [reply sass/themes/cr/2019/components/full-height-single-image-single-copy/_full-height-single-image-single-copy.scss]: 😆 overcompensating

Comment (AndyEPhipps) [reply sass/themes/cr/2019/components/full-height-single-image-single-copy/_full-height-single-image-single-copy.scss]: 👍

**Pull request merged**